openwisp_users/accounts/templates/account/email/base_message.html

```diff
@@ -0,0 +1 @@
+{% extends "openwisp_utils/email_template.html" %}
```

Comment:
here we could avoid extending from this template and either leave this empty or remove it